<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaConfiguracao</class>
 <widget class="QDialog" name="JanelaConfiguracao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1911</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BotaoCancelar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Canelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Aplicar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>User anilist</string>
   </property>
  </widget>
  <widget class="QListWidget" name="DiretorioAnimesLista">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DiretorioAnimesAdiciona">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DiretorioAnimesRemove">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BotaoAutorizar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
</string>
   </property>
   <property name="text">
    <string>Autorizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diretorio de animes</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="anilistUsername">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayMensagem">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold; 
color: rgb(220, 220, 220);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
